docker/Dockerfile.core-services

```diff
@@ -0,0 +1,197 @@
+FROM centos:7
+
+ARG hoot_user
+ARG hoot_uid
+ARG hoot_gid
+ARG hoot_group
+ARG hoot_shell=/bin/bash
+ARG hoot_home
+
+ARG glpk_version=4.64
+ARG liboauthcpp_version=0.1.0
+ARG libphonenumber_version=8.12.39
+ARG libpostal_version=1.1
+ARG node_version=14.16.1
+ARG npm_version=6.14.12
+ARG v8_version=8.4.371.19
+ARG stxxl_version=1.3.1
+ARG armadillo_version=10.8.2
+ARG gdal_version=3.2.3
+ARG geos_version=3.9.3
+ARG libgeotiff_version=1.6.0
+ARG proj_version=7.2.1
+ARG devtoolset_version=8
+ARG v8_version=8.4.371.19
+ARG stxxl_version=1.3.1
+ARG postgresql_version=14
```

Comment:
Is there a way of combining the values found [here](https://github.com/ngageoint/hootenanny/blob/master/VagrantProvisionVars.sh) with the above versions?  Also if you notice in that file, Hoot is being built and tested against Postgres 13 not 14.  Probably not a big deal but it is already out of sync.

Reply:
:bulb: good idea
I can look into it

Reply:
We also have some versions here https://github.com/ngageoint/hootenanny-rpms/blob/master/config.yml